src/anomalib/models/dfm/lightning_model.py

```diff
@@ -24,9 +24,9 @@ class Dfm(AnomalyModule):
 
     Args:
     ----
+        input_size (tuple[int, int]): Input size for the model.
```

Comment:
Should we also add a default value for the `input_size`? Is this specified in the paper?

Reply:
We override the input size with that of the dataloader. And, if someone uses the API and sets the dataloader input size to a different value then it will throw an error.

Reply:
yeah, they need to explicitly specify the size to avoid issues. But this is the current way to create the datamodule and the model via API, right?

```python
from anomalib.data import Visa
from anomalib.models import EfficientAd

datamodule = Visa(image_size=(256, 256), ...)
model = EfficientAd(input_size(256, 256))
...
```